src/openssl.patch/debian.patch/40-disable-test-cases-with-fips-enabled.patch

```diff
@@ -0,0 +1,160 @@
+diff --git a/test/recipes/05-test_rand.t b/test/recipes/05-test_rand.t
+index 3ae254031c..95ebba15f0 100644
+--- a/test/recipes/05-test_rand.t
++++ b/test/recipes/05-test_rand.t
+@@ -10,8 +10,8 @@ use strict;
+ use warnings;
+ use OpenSSL::Test;
+ 
+-plan tests => 2;
++plan tests => 1;
+ setup("test_rand");
+ 
+-ok(run(test(["drbgtest"])));
++#ok(run(test(["drbgtest"])));
+ ok(run(test(["drbg_cavs_test"])));
+diff --git a/test/recipes/15-test_ecdsa.t b/test/recipes/15-test_ecdsa.t
+deleted file mode 100644
+index 82a85594c3..0000000000
+--- a/test/recipes/15-test_ecdsa.t
++++ /dev/null
+@@ -1,12 +0,0 @@
+-#! /usr/bin/env perl
+-# Copyright 2015-2016 The OpenSSL Project Authors. All Rights Reserved.
+-#
+-# Licensed under the OpenSSL license (the "License").  You may not use
+-# this file except in compliance with the License.  You can obtain a copy
+-# in the file LICENSE in the source distribution or at
+-# https://www.openssl.org/source/license.html
+-
+-
+-use OpenSSL::Test::Simple;
+-
+-simple_test("test_ecdsa", "ecdsatest", "ec");
+diff --git a/test/recipes/15-test_mp_rsa.t b/test/recipes/15-test_mp_rsa.t
+index 9271dba042..255bbbe240 100644
+--- a/test/recipes/15-test_mp_rsa.t
++++ b/test/recipes/15-test_mp_rsa.t
+@@ -17,9 +17,9 @@ use OpenSSL::Test::Utils;
+ 
+ setup("test_mp_rsa");
+ 
+-plan tests => 31;
++plan tests => 30;
+ 
+-ok(run(test(["rsa_mp_test"])), "running rsa multi prime test");
++#ok(run(test(["rsa_mp_test"])), "running rsa multi prime test");
+ 
+ my $cleartext = data_file("plain_text");
+ 
+diff --git a/test/recipes/30-test_engine.t b/test/recipes/30-test_engine.t
+deleted file mode 100644
+index 03c96cde09..0000000000
+--- a/test/recipes/30-test_engine.t
++++ /dev/null
+@@ -1,18 +0,0 @@
+-#! /usr/bin/env perl
+-# Copyright 2015-2016 The OpenSSL Project Authors. All Rights Reserved.
+-#
+-# Licensed under the OpenSSL license (the "License").  You may not use
+-# this file except in compliance with the License.  You can obtain a copy
+-# in the file LICENSE in the source distribution or at
+-# https://www.openssl.org/source/license.html
+-
+-
+-use strict;
+-use warnings;
+-
+-use OpenSSL::Test;
+-
+-setup("test_engine");
+-
+-plan tests => 1;
+-ok(run(test(["enginetest"])), "running enginetest");
+diff --git a/test/recipes/30-test_evp.t b/test/recipes/30-test_evp.t
+index 2385105b8b..c0b792d903 100644
+--- a/test/recipes/30-test_evp.t
++++ b/test/recipes/30-test_evp.t
+@@ -14,7 +14,7 @@ use OpenSSL::Test qw/:DEFAULT data_file/;
+ 
+ setup("test_evp");
+ 
+-my @files = ( "evpciph.txt", "evpdigest.txt", "evpencod.txt", "evpkdf.txt",
```

Comment:
Removing evpciph.txt is a step further than necessary. You can just remove tests for AES-GCM with IV length != 12 bytes (lines 780-804, 838-862, 896-920, and 960-984 last I checked)

Reply:
Fixed, thanks.